app/api/contests/enter.go

```diff
@@ -95,6 +95,7 @@ func (srv *Service) enter(w http.ResponseWriter, r *http.Request) service.APIErr
 				itemInfo.ContestParticipantsGroupID, qualificationState.groupID,
 				itemInfo.Now, itemInfo.Duration, totalAdditionalTime).Error())
 			service.MustNotBeError(store.GroupGroups().After())
+			service.MustNotBeError(store.Attempts().ComputeAllAttempts())
```

Comment:
Should add a comment explaining the reason you need to recompute (i.e., what may be created/updated)

Reply:
fixed